website/docs/d/lb_app_profile.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_lb_app_profile"
+sidebar_current: "docs-vcd-data-source-lb-app-profile"
+description: |-
+  Provides a NSX load balancer application profile data source.
```

Comment:
Grammar nit: should be "an NSX", because pronouncing "an en-es-ex".

Should we also mention Edge Gateway here?

Reply:
`Provides a NSX edge gateway load balancer application profile data source.`?

Reply:
Sure!

Reply:
Changed.